packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -281,14 +281,21 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
     schemaComposer.Query.addFields(queryFields)
   }
 
-  protected createStandardTypes(schemaComposer: SchemaComposer<any>) {
-    schemaComposer.add(IDScalarInputType)
-    schemaComposer.add(StringScalarInputType)
-    schemaComposer.add(IntScalarInputType)
-    schemaComposer.add(FloatScalarInputType)
-    schemaComposer.add(BooleanScalarInputType)
-    schemaComposer.add(SortDirectionEnum)
-    schemaComposer.add(PageRequest)
+  private createStandardInputTypes(schemaComposer: SchemaComposer<any>, schema: GraphQLSchema) {
+    schemaComposer.add(PageRequest);
+    schemaComposer.add(IDScalarInputType);
+    schemaComposer.add(SortDirectionEnum);
+    schemaComposer.add(StringScalarInputType);
+    schemaComposer.add(BooleanScalarInputType);
+    schemaComposer.add(createInputTypeForCustomScalar(GraphQLInt));
```

Comment:
Are we using `createInputTypeForCustomScalar` for Float and Int because the values are the same? If so, maybe `createInputTypeForCustomScalar` is a misleading name.

Reply:
How about `createInputTypeForScalar`